ql/src/test/queries/clientpositive/lateral_view.q

```diff
@@ -10,6 +10,11 @@ EXPLAIN SELECT myTable.* FROM src LATERAL VIEW explode(array(1,2,3)) myTable AS
 EXPLAIN SELECT myTable.myCol, myTable2.myCol2 FROM src LATERAL VIEW explode(array(1,2,3)) myTable AS myCol LATERAL VIEW explode(array('a', 'b', 'c')) myTable2 AS myCol2 LIMIT 9;
 EXPLAIN SELECT myTable2.* FROM src LATERAL VIEW explode(array(array(1,2,3))) myTable AS myCol LATERAL VIEW explode(myTable.myCol) myTable2 AS myCol2 LIMIT 3;
 
+EXPLAIN CBO SELECT * FROM src LATERAL VIEW explode(array(1,2,3)) myTable AS myCol SORT BY key ASC, myCol ASC LIMIT 1;
```

Comment:
It is a good idea to verify the `EXPLAIN CBO` plan in the presence of lateral views. I am wondering if it would be better to identify a few representative queries and then put the EXPLAIN CBO statements in a new .q file (e.g., `cbo_lateral_views.q`) going from the simpler to the more complicate queries. These would make testing more explicit and it would also reduce the changes in existing .q.out files leading to fewer merge conflicts when backporting. WDYT?

Reply:
I actually thought about doing it this way, but I think it makes sense to always run a CBO plan with any query that we run.  My personal opinion is that we should add this to the framework and run it by default without having to explciitly specify it, but that's a change for another day.

Reply:
Done